content/blog/2021/2021-02-10-explaining-models.md

```diff
@@ -0,0 +1,53 @@
+longExcerpt = "New Orange Explain add-on for understanding predictions and predictive models."
+x2images = false  # true if using retina screenshots, else false
++++
+
+It is easy to build powerful predictive models in Orange. But how does the model "look like"? Which attributes and which values of those attributes are important? And when making predictions, which attributes contributed to the decision? **Orange's new Explain add-on** helps you answer all those questions.
+
+Related: {{< link_new url="/blog/2019/2019-11-20-belgrade-workshop/" name="Explaining Models">}}
+
+Go to **Options --> Add-ons** and install Explain add-on. Restart Orange for the add-on to appear. It only contains two widgets, but boy are they great!
+
+Let us start with the attrition data set from the Datasets widget. We will go with **Attrition - Train**, which a data set on which employees resigned from the company and which stayed. The target variable is called Attrition, where **No** means that the employee stayed and **Yes** that the employee resigned. The other attributes describe the employee - her position, education, department, years since promotion, and so on.
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-data-table.png" %}}
+
+Next, we will build a simple logistic regression predictive model. If inspecting the model in Test and Score, we learn that the model has an AUC of 0.788 and 86 % CA. But what *kind* of a model is this? How does it makes its decisions?
+
+Let us add Explain Model to Logistic Regression and add another connection passing the data. The workflow should look like this:
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-model-workflow.png" %}}
+
+Now open Explain Model. The widget lists top ranked variables, which means they contribute the most to the selected target variable. As we are trying to understand why people leave the company, we have set the target variable to Yes.
+
+The highest ranked variable is OverTime - this is the variable with the highest impact on the prediction. Having a high value of overtime (red dots on the right) means the employee is likely to quit. Also, having low job satisfaction contributes to attrition (blue values on the right). The visualization shows the values that have a high impact on the model on the right and those with a low impact on the left. The color of dot represents the value of the attribute (red for higher values and blue for lower).
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-explain-model.png" %}}
+
+Let us look at, for example, YearsAtCompany. How would we interpret this? The variable has more dots on the left, which means it contributes against the target value (against attrition, i.e. employees will stay). Red dots refer to the value of the attribute. So if the employee is with the company for a long time (high value == red dot), it means it is more likely she will stay (dots are on the left, while our target value is Yes).
```

Comment:
> The variable has more dots on the left, which means it contributes against the target value (against attrition, i.e. employees will stay)

It is difficult to say that there are more values on the left (they are just further away from the center). Also, it is not so much important how many values are on the left and on the right we are more interested in what values (high - low) contribute in which direction. I would change this sentence to something like:

`The variable has more red dots on the left, which means high variable values contributes against the target value (against attrition, i.e. employees will stay)`